packages/steppp/src/index.ts

```diff
@@ -250,8 +212,10 @@ function Steppp(element: HTMLElement, options: any = defaultOptions): Instance {
     height?: number
   ): number => {
     element.style.height = "";
+    const originalDisplay = step.style.display;
```

Comment:
I'm not sure how this didn't bite us before. I'm temporarily making the element visible (between repaints) in order to measure the height, but I wasn't resetting the original display property after performing that measurement. This fixes that and should prevent any odd surprises from coming up.